src/Views/AbstractInstallerView.vala

```diff
@@ -19,6 +19,7 @@
 public abstract class AbstractInstallerView : Gtk.Grid {
     public bool cancellable { get; construct; }
     public unowned Gtk.Widget? previous_view { get; set; }
+    public Gtk.Label? test_label = null;
```

Comment:
Not needed anymore since we're not setting a timer. Can just be defined in the construct block like before.